WORKSPACE

```diff
@@ -109,3 +109,36 @@ dependency_support()
 load("//:init.bzl", "init")
 
 init(python_interpreter_target = "@rules_hdl_cpython//:install/bin/python3")
+
+# containers
+http_archive(
+    name = "io_bazel_rules_docker",
+    sha256 = "4349f2b0b45c860dd2ffe18802e9f79183806af93ce5921fb12cbd6c07ab69a8",
+    strip_prefix = "rules_docker-0.21.0",
+    urls = ["https://github.com/bazelbuild/rules_docker/releases/download/v0.21.0/rules_docker-v0.21.0.tar.gz"],
+)
+load(
+    "@io_bazel_rules_docker//repositories:repositories.bzl",
+    container_repositories = "repositories",
+)
+container_repositories()
+
+load("@io_bazel_rules_docker//repositories:deps.bzl", container_deps = "deps")
+container_deps()
```

Comment:
I think you want to move all the init specific stuff to a bzl file in the dependency_support folder, and call it in the root init.bzl file of this repo. Following the example of boost and py_bind.